pyaedt/edb_core/EDB_Data.py

```diff
@@ -3547,6 +3548,15 @@ def use_radiation_boundary(self, value):  # pragma: no cover
         if isinstance(value, bool):
             self._use_radiation_boundary = value
 
+    @property
+    def output_aedb(self):  # pragma: no cover
```

Comment:
This property is easy to Unit Test. Please remove the Pragma comment and add a UT for getter and setter.
Please also add a docstring (to property only)

Reply:
done